pkgs/by-name/th/thrift-ls/package.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "thrift-ls";
+  version = "0.2.1";
+
+  src = fetchFromGitHub {
+    owner = "joyme123";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-Qib/xbTQiR0VpHsCOt5HGkbytaQ0GSCqaHVYFNPYGVs=";
+  };
+
+  vendorHash = "sha256-YoZ2dku84065Ygh9XU6dOwmCkuwX0r8a0Oo8c1HPsS4=";
+
+  postInstall = ''
+    mv $out/bin/thrift-ls $out/bin/thriftls
```

Comment:
Could you explain why you're renaming the binary? I don't really have an issue with it, but wondering about the motivation.

Reply:
Renaming the binary to match behavior of upstream. 
If we don't change the name then the binary will work for users with nvim lsp-config

Reply:
I'll leave ownership to you though as I am not a maintainer

Reply:
Yep makes sense - go for it :smile: